### ISSUE TITLE
Declaring constant in namespaced file without classes generates unsolvable sniff failure

### ISSUE THREAD

**Issue by rdohms** [slevomat/coding-standard#437]
Considering this file:

```
<?php
declare(strict_types=1);

namespace Some\Ns;

const HELLO = 'world';

echo HELLO;
```
Generates the following error:
`Constant HELLO should not be referenced via a fallback global name, but via a use statement.`

Running phpcbf does this which breaks it even further:
```
<?php
declare(strict_types=1);

namespace Some\Ns;

use const HELLO;

const HELLO = 'world';

echo HELLO;
```

This makes the definition broken due to "already defined" and the `use` statement is flagged as not needed.

What do you think about this?

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/7592b77d33a9e1880089df2eb50f51a1c7e2a70b
